src/components/startPage/start.js

```diff
@@ -0,0 +1,99 @@
+////[[.................. Creating HTML structure............................]]////
+
+const body = document.querySelector("body");
+const main = document.createElement("main");
+body.appendChild(main);
+
+//creating left section
+const siteSection = document.createElement("div");
+siteSection.classList.add("siteSection");
+main.appendChild(siteSection);
+const buttonContainer = document.createElement("div");
+
+///Creating left section inside elements
+
+//creating logo
+
+const img = document.createElement("img");
+siteSection.appendChild(img);
+img.classList.add("logo");
+img.setAttribute(
+  "src",
+  "/src/components/startPage/img/toppng 7.png"
+);
+img.setAttribute("alt", "Game of Thrones logo");
+
+//creating poster
+
```

Comment:
You should not create logo component its not yours task.

Reply:
I guess i misanderstood the tasks, i thought i should create whole main page.
I rewrite this